client/my-sites/domains/components/mapping-instructions/index.jsx

```diff
@@ -30,13 +30,115 @@ class DomainMappingInstructions extends React.Component {
 		areDomainDetailsLoaded: PropTypes.bool,
 		domainName: PropTypes.string,
 		isAtomic: PropTypes.bool,
+		wpcomDomainName: PropTypes.string,
 	};
 
 	static defaultProps = {
 		areDomainDetailsLoaded: false,
 		isAtomic: false,
 	};
 
+	renderNameServerInstructions() {
+		const { translate } = this.props;
+
+		const nameServerInstructionsMessage = translate(
+			'Please log into your domain name registrar account and {{strong}}update the name servers{{/strong}} of your domain to use the following values, as detailed in {{link}}these instructions{{/link}}:',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Please log into your account at your domain registrar and {{strong}}update the name servers{{/strong}} of your domain to use the following values, as detailed in {{link}}these instructions{{/link}}:' )` *ES Score: 9*